test/gerkin/features/steps/db_read.js

```diff
@@ -0,0 +1,79 @@
+const pactum = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { header, localhost } = require('./helpers/helpers');
+
+let specDataRead;
+
+const baseUrl = `${localhost}database/{id}`;
+  
+Before(() => {
+  specDataRead = pactum.spec();
+});
+
+// Background: The user wants to retrieve the Digital Registries database information with schema versions
+Given(
+  'The user wants to retrieve the Digital Registries database information with schema versions', () => {
+  return 'The user wants to retrieve the Digital Registries database information with schema versions';
+  }
+);
+
+// Scenario: The user successfully receives one Digital Registries database information with schema versions
+When(
+  'The user triggers an action with a valid payload to display one database schema versions', () => {
+    specDataRead
+      .get(`${baseUrl}`)
+      .withHeaders(`${header.key}`, `${header.value}`)
+      .withPathParams('id', 353);
+  }
+);
+
+Then('The user received one database information with schema versions', async () => {
+  await specDataRead.toss();
+  specDataRead.response().should.have.status(200);
```

Comment:
Is the response from API empty or there is some json or string ? If yes, check if those response is what you expect :)